### PR TITLE
framework/st_things : logic cleanup

### PR DESCRIPTION
**Pull request by JeongJunSik**
- remove pipe logic
- change typing error

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check Result:
framework/src/st_things/things_stack/easy-setup/easysetup_manager.c:238: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/st_things/things_stack/easy-setup/easysetup_manager.c:238: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/st_things/things_stack/easy-setup/easysetup_manager.c:238: ERROR: [IDT_M_TAB] please, no spaces at the start of a line

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c [hunk 1]
```diff
@@ -235,34 +235,16 @@ esm_result_e esm_init_easysetup(int restart_flag, things_server_builder_s *serve
...
-			}
-			if (ci_token_expire_fds[1] != -1) {
-				close(ci_token_expire_fds[1]);
-				ci_token_expire_fds[1] = -1;
-			}
-#else
+ 			pthread_cancel(gthread_id_cloud_refresh_check);
 			pthread_join(gthread_id_cloud_refresh_check, NULL);
-#endif
+			pthread_detach(gthread_id_cloud_refresh_check);
```
Comment (sunghan-chang) [inline framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: Don't use both, pthread_join and pthread_detach. You shoule use only one function among them.

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c [hunk 2]
```diff
@@ -168,7 +168,7 @@ int esm_set_device_property_by_app(char *name, const wifi_mode_e *mode, int ea_m
 
 	int i = 0;
 
-	if (mode == NULL || ea_mode < 1 || freq >= WiFi_FREQ_EOF) {
+	if (mode == NULL || ea_mode < 1 | freq < WiFi_24G || WiFi_FREQ_EOF <= freq) {
```
Comment (sunghan-chang) [inline framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: I think readabilty is not good.
```var < XX || var > YY``` is better I think.

**Comment by sunghan-chang** [comment]
Please check violations of coding rule

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check Result:
framework/src/st_things/things_stack/utils/things_ping.c:207: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
framework/src/st_things/things_stack/utils/things_ping.c:1069: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/st_things/things_stack/utils/things_ping.c:1126: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
framework/src/st_things/things_stack/utils/things_ping.c:1259: ERROR: [SPC_M_KWD] space required before the open parenthesis '('

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c (hunk 1 above)
Comment (JeongJunSik) [reply framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: Ok. remove pthread_detach

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c (hunk 2 above)
Comment (JeongJunSik) [reply framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: OK. i chaged

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**